webpack.config.prod.js

```diff
@@ -64,7 +59,20 @@ export default {
     new webpack.optimize.DedupePlugin(),
 
     // Minify JS
-    new webpack.optimize.UglifyJsPlugin()
+    new webpack.optimize.UglifyJsPlugin(),
```

Comment:
I also had to add an options object to the UglifyJsPlugin constructor for the build to succeed.
```
    new webpack.optimize.UglifyJsPlugin({
      sourceMap: true
    }),
```
Per [the docs](https://webpack.js.org/guides/migrating/#uglifyjsplugin-sourcemap)